notebooks/standardize_events_data.ipynb

```diff
@@ -22,37 +34,234 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 2,
    "id": "f8f05b8e-213b-47ef-9189-486d2e702267",
    "metadata": {},
    "outputs": [],
    "source": [
-    "import pandas as pd\n",
-    "# Define the GCS path to the Parquet file\n",
-    "gcs_path = 'https://storage.googleapis.com/derisk-persistent-state/zklend_data/events_sample.parquet'\n",
+    "# Dictionary of Parquet URLs\n",
```

Comment:
Pls use the same method for loading data as here: https://github.com/CarmineOptions/derisk-research/blob/master/notebooks/Liquidators_data.ipynb (cell 2). No need to use `gcsfs`.